wordle-web/src/scripts/wordsService.ts

```diff
@@ -7,9 +7,105 @@ export abstract class WordsService {
     return this.#words.includes(word)
   }
 
-  static validWords(): Array<string> {
-    //Todo
-    return new Array<string>()
+  static getAll() {
+    let all = Array()
+    for (const word of this.#words) {
+      all.push(word)
+    }
+    return all
+  }
+
+  static currentWords(curGuess: string[], currentList: string[]) {
+    //key:
+    //  letter[0] = char
+    //  letter[1] = status
+    //  letter[2] = index
+    let wordList = Array()
+    //let wordList = this.#words
+    for (const word of currentList) {
+      let add = true
+      let split = word.split('')
+      for (let i = 0; i < curGuess.length; i++) {
+        if (split[i] != curGuess[i]) {
+          add = false
+        }
+      }
+      if (add == true) {
+        wordList.push(word)
+      }
+    }
+    /*console.log("PressCheck")
```

Comment:
add unit tests and remove commented logs

Reply:
I'll get to unit tests as soon as I finish the rest of the assignment (and probably implement that code you mentioned above to simplify things).  Also, I forgot how many test lines of code I left in there, thanks for calling those out, I'll need to go back and see how many more I forgot in.